packages/grid/docker-compose.pull.yml

```diff
@@ -7,7 +7,7 @@ services:
   #   image: rabbitmq:${RABBITMQ_VERSION?Variable not Set}${RABBITMQ_MANAGEMENT:-}
 
   seaweedfs:
-    image: "${DOCKER_IMAGE_SEAWEEDFS?Variable not set}:${SEAWEEDFS_VERSION}"
+    image: "${DOCKER_IMAGE_SEAWEEDFS?Variable not set}:${VERSION-latest}"
```

Comment:
Q: Should it be `SEAWEEDFS_VERSION` instead of latest ?

Reply:
We're pulling from dockerhub openmined, and therefore using the same version as backend or frontend.